config.json

```diff
@@ -1182,6 +1182,14 @@
         "dependencyOnly": false,
         "templateSet": "rxjava"
       },
+      {
```

Comment:
There is already an existing `com.xamarin.androidx.migration` for `1.0.8`.  Should we update that one instead of adding an additional entry?